docs/02-requirements/LZ-02-01.adoc

```diff
@@ -7,44 +7,38 @@ Architekten können Stakeholder und deren Anliegen sowie deren
 Auswirkungen auf die Softwarearchitektur oder den Entwurfs- und
 Entwicklungsprozess identifizieren. (R2)
 
-* Beispiele für Stakeholder und ihre Anliegen sind wie folgt:
+* Beispiele für Stakeholder und ihre Anliegen:
 
 |===
 |Interessengruppe |Anliegen der Interessengruppe
 
 | Produktmanagement
 | z. B. benötigte Zeit für die Umsetzung der Anforderungen
 
-| Projektmanager
+| Projektmanagement
 | z. B. benötigte Zeit und Budget für die Umsetzung, verbundene Risiken des
 gewählten architekturellen Ansatzes
 
-| Requirements Ingenieure
+| Requirements Engineering, Fachbereiche
 | z. B. Erfüllung der Anforderungen
 
-| Entwickler
+| Entwicklungsteam
 | z. B. zu implementierende Komponenten und Schnittstellen, Protokolle,
 technologische Einschränkungen
 
-| Qualitätssicherung und Tester
+| Qualitätssicherung und Test
 | z. B. isoliertes Testen von Komponenten
 
 | Betrieb
 | z. B. Infrastrukturanforderungen im Zusammenhang mit dem Betrieb des Systems
 
 |===
 
-Softwarearchitekten können potenzielle Konflikte zwischen kurz- und
-langfristigen Zielen erklären, um mögliche Konflikte (z.B., Geschäfts- und
-Projektziele vs. Architektur- und Wartbarkeitsziele) zu lösen. (R1)
+Softwarearchitekt:innen können potenzielle Konflikte zwischen kurz- und langfristigen Zielen erklären, um mögliche Konflikte (z.B., Geschäfts- und Projektziele vs. Architektur- und Wartbarkeitsziele) zu lösen. (R1)
```

Comment:
Personally I would prefer the longer form "Softwarearchitektinnen und -architekten" for quite a number of reasons which explanation would lead to a deep rabbit hole of side-quests. But I can live with that.

Reply:
It is the default of iSAQB and should be like this in all curricula. 🙂